### ISSUE TITLE
What's with create 0.5.1???

### ISSUE THREAD

**Issue by Hereux** [Rabbitminers/Extended-Flywheels-Fabric#5]
It crashes due to incompability. I can't fix it myself because I can't program with Java. I think there are some other people who would like to use the mod again, so how about a life sign from you devs?

**Comment by Hereux** [comment]
And it's not like I don't want to help, I'm willing to help as good as I can, but without knowing Java the only thing I could do is translating the mod into german.

**Comment by ebaystudio** [comment]
Yeah the dev isnt that great at posting updates. I think you have to go to his Discord to get any news.
